src/ansi-c/expr2c.cpp

```diff
@@ -3749,6 +3778,8 @@ std::string expr2ct::convert_with_precedence(
       return convert_statement_expression(src, precedence=15);
     else if(statement==ID_gcc_builtin_va_arg_next)
       return convert_function(src, "gcc_builtin_va_arg_next", precedence=16);
+    else if(statement == ID_cpp_new_array)
+      return convert_cpp_new_array(src, precedence = 16);
```

Comment:
Shouldn't this go in `expr2cpp`?

Reply:
This is still open -- the ansi-c pretty printer isn't meant to be the "catch all" any more.